ads/mywidget.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {loadScript, validateData} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function mywidget(global, data) {
+  validateData(data, ['cid']);
+
+  const height = data.height;
+  let isReady = false;
+
+  if (!(height >= 0)) {
```

Comment:
`height < 0` would not quite fit :) `data.height` can be undefined here, if attribute has invalid value or is absent, so... I can change condition to: `if (typeof height === 'undefined' || height < 0)` if you like. But there is no guarantee, that the default value of absent attribute not will be `null` tomorrow :)